README.md

```diff
@@ -118,44 +118,45 @@ Commands:
 ```
 
 ## Customizable Template
-You can use this example "recipe" to quickly customize the sandbox to your development environment. Copy the code below and paste it into a text editor.
+You can use this code sample to quickly customize the sandbox to your development environment. Copy the code below and paste it into a text editor.
 
 ```
-{  
-    "sandbox":{  
+{
+    "sandbox":{
         "clonable":true,
-        "properties":[  
-            {  
+        "properties":[
+            {
                 "property":"123456:2",
-                "requestHostnames":[  
+                "requestHostnames":[
                     "localhost2"
                 ],
-                "cpcode": 1234
```

Comment:
I would rather like to have spaces here and in all other places (`"key": value`, not `"key":value`), it makes it more readable, less cluttered.